db/schema.rb

```diff
@@ -34,6 +32,8 @@
     t.datetime "created_at",                                   null: false
     t.datetime "updated_at",                                   null: false
     t.string   "email_alert_signup_content_id"
+    t.text     "organisation_content_ids",      default: [],                array: true
```

Comment:
Dropped in a previous commit of this PR

https://github.com/alphagov/policy-publisher/commit/5ff57d45e967bc18225b3fbaa622f38ad842e0b4#diff-8a1923c466f71d93472de30d1c687a83R4

This commits puts it back in.